docs/e2e.md

```diff
@@ -0,0 +1,152 @@
+<!--
+SPDX-FileCopyrightText: The RamenDR authors
+SPDX-License-Identifier: Apache-2.0
+-->
+
+# End to End
+
+RamenDR end-to-end (e2e) tests validate various scenarios for regional DR
+using predefined workloads and deployment methods.
+
+## Prepare `config.yaml` File
+
+Create a `config.yaml` file by copying the `config.yaml.sample` template:
+
+```sh
+cat e2e/config.yaml.sample >> e2e/config.yaml
+```
+
+Update `config.yaml` by adding kubeconfig paths of the clusters, with format:
+
+```yaml
+Clusters:
+    c1:
+        kubeconfigpath: /path/to/kubeconfig/c1
+    c2:
+        kubeconfigpath: /path/to/kubeconfig/c2
+    hub:
+        kubeconfigpath: /path/to/kubeconfig/hub
+```
+
+If `drenv` was used to configure RDR clusters, easily copy `config.yaml.sample`
+and add the clusters kubeconfig paths into `config.yaml` using:
+
+```sh
+cat e2e/config.yaml.sample ~/.config/drenv/rdr/config.yaml >> e2e/config.yaml
+```
+
+## Run All E2E Tests
+
+`run.sh` is a shell script that runs the RamenDR E2E tests for regional DR.
+
+To run all the tests:
+
+```sh
+cd e2e
+./run.sh
+```
```

Comment:
We don't show the cd command for all example, and it will be wrong to show them since the cd command works only when you are in the ramen root directory. We have 2 ways to handle this:

Add a note at the start that all commands run from the e2e directory

> [!IMPORTANT]
> All commands must run from the e2e directory


Or show how to run everything from the root directory:

```
(cd e2e && ./run.sh)
```

I always run from the e2e directory, since it nicer to use both for development of e2e and for using it.

Reply:
Added Important alert with description in "Running End to End tests" section.